app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -81,7 +82,8 @@
 				new AddBabyChangingTable(o),
 				new AddFireHydrantType(o),
 				new AddParkingType(o),
-        		new AddPowerPolesMaterial(o)
+        new AddPowerPolesMaterial(o),
+        new AddCarWashType(o),
```

Comment:
FYI: In the root directory of the project, there is a code_scheme.xml. You can apply this in Android Studio, so that the IDE will automatically adapt the code style (indentation etc.).